usaspending_api/etl/es_award_template.json

```diff
@@ -0,0 +1,442 @@
+                        {
+  "index_patterns": ["*awards*"],
+  "settings": {
```

Comment:
The settings should match the transactions template and the null values be overwritten in the same manner.
```json
  "index_patterns": null,
  "settings": {
    "index.mapping.ignore_malformed": true,
    "index.max_result_window": null,
    "index.refresh_interval": -1,
    "index": {
      "number_of_shards" : 5,
      "number_of_replicas" : 0
    },
```